app/wayfinders/wayfinder.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+class Wayfinder < BaseWayfinder
```

Comment:
Please add class doc noting that this is the wayfinder factory slash default wayfinder for anything that doesn't define its own